src/QMCWaveFunctions/detail/SYCL/sycl_determinant_helper.cpp

```diff
@@ -0,0 +1,88 @@
+//////////////////////////////////////////////////////////////////////////////////////
+// This file is distributed under the University of Illinois/NCSA Open Source License.
+// See LICENSE file in top directory for details.
+//
+// Copyright (c) 2019 QMCPACK developers.
```

Comment:
2022?

Reply:
These files are started by copying the CUDA conterpart, thus carries the old year. Since files are updated, the year should be 2022 now. I updated all the related files.